### ISSUE TITLE
Screen reader focus sometimes jumps off of input field and up to page header

### ISSUE THREAD

**Issue by melanie-exygy** [civiform/civiform#5429]
**Describe the bug**
In the applicant form, when focus is on an input field, the screen reader will sometimes shift focus to the page header.  This is inconsistent.  Also, visual focus seems to stay on the input field.

**To Reproduce**

1. On staging, clear the database and make sure the default programs are published.
2. From the home page, under "Minimal Sample Program", click on 'Apply'.
3. On the summary page, turn on your screen reader.
4. Click "Answer" on the row with the question "Please enter your first and last name"
5. Note that the black focus box is around the main content of the page rather than the name question input.  Also, note that the screen reader begins with "heading level 1...".  Interestingly, the cursor does stay on the first name input field.

**Expected behavior**
The screen reader should begin reading the name question text rather than the page-level header.  The focus box should be on the first name input field.

**Desktop (please complete the following information):**

- OS:  iOS
- Browser Chrome
- Screen reader:  VoiceOver

**Additional context**
Here are some suggested steps for the developer to try:
- Try with other browsers (Safari, Firefox, Edge)
- Try with other screen readers (Use BrowserStack to try JAWS and NVDA)
- Try with optional vs. required questions.
- Try with a previously answered question and one that has not been previously answered.
- Try on single error validation with this commit [1fd978cd354d7ab9af10f89f50c70c19c37f0233](https://github.com/civiform/civiform/commit/4ab0860962e8cbde957afb7028a854050ff7a7f7), which is before autofocus was added to selected input fields.
- See if render order could be causing the problem.  Test with some `console.log` statements.
- Read this [spec](https://html.spec.whatwg.org/multipage/interaction.html#autofocus-candidates)

**Comment by bion** [comment]
According to our a11y experts this is a bug either in Chrome or VoiceOver and not something we can address with our code. WAI when tested using other browsers and screen readers.
